src/template/js/sdk/Transport/mock.js

```diff
@@ -79,11 +79,12 @@ function receive(_callback) {
 }
 
 function event(module, event, value) {
-  const id = Object.entries(event_map).find(([k, v]) => v === module.toLowerCase() + '.' + event.toLowerCase())[0]
-  if (id) {
+  const eventMap = event_map
```

Comment:
This fixed a weird scoping problem. event_map is a statically loaded module so it gets stored in this method scope statically.